### DISCUSSION TITLE
Bitbucket support

### DISCUSSION THREAD

**Discussion by acheonline** [jreleaser/jreleaser#442] [Q&A]
Hi guys! First of al I would say - thank you for so helpful solution. And my question is - do you have any support/instructions for jreleaser and bitbucket interaction? I would like to use your solution, but we have a bitbucket

**Comment by aalmiray** [comment]
Thank you 😄 

As far as I understand BitBucket does not offer release options as those found in GitHub/GitLab/Gitea that is, a page where you get

 - a release name
 - body (could be changelog, release notes, any free text)
 - release assets (files linked to the release)

For this reason explicit BitBucket support is yet forthcoming (see #12). If there's a convention provided by BitBucket for hosting a release page + assets then we can follow it, assuming of course there's a REST API that we can leverage to automate the whole shebang.

In the meantime you may use the [Generic Git](https://jreleaser.org/guide/latest/configuration/release/generic.html) option which will only let you tag a release but does not create a release page. Assets may be uploaded to JFrog Artifactory, AWS S3 (coming up in v0.8.0), or an HTTP/HTTPS server under your control.
